src/commands/__tests__/log.test.js

```diff
@@ -35,27 +40,72 @@ describe(testContext(__filename), function () {
         })
     })
 
-    // TODO: enable this test once APIs are ready
-    it('notifies that the retrospective survey being loaded when requested' /* , function () {
-      // nock('https://game.learnersguild.test')
-      //   .post('/graphql')
-      //   .reply(200, {data: {createdIds: ['00000000-1111-2222-3333-444444444444']}})
+    it('prints all the queations when not given a number', function () {
+      nock('https://game.learnersguild.test')
+        .post('/graphql')
+        .reply(200, {data: {
+          getRetrospectiveSurvey: {
+            questions: [
+              {
+                id: '99ede319-882f-4dc8-81e2-be43f891a111',
+                subjectType: 'player',
+                responseType: 'text',
+                responseIntructions: 'these are the instructions',
+                body: 'this is the question body',
+                subject: {
+                  id: '34278883-2e76-42b6-a8aa-fa74a1892f90',
+                  name: 'Rosemarie Kub',
+                  handle: 'mobile81'
+                }
+              },
+              {
+                id: '99ede319-882f-4dc8-81e2-be43f891a122',
+                subjectType: 'player',
+                responseType: 'text',
+                responseIntructions: 'these are the instructions',
+                body: 'this is the second question body',
+                subject: {
+                  id: '34278883-2e76-42b6-a8aa-fa74a1892faa',
+                  name: 'Trevor Little',
+                  handle: 'tlittle'
+                }
+              },
+            ]
+          }
+        }})
 
       const {lgJWT, lgPlayer} = this
-      this.invoke(['-r'], this.notify, {lgJWT, lgPlayer})
-      expect(this.notifications[0]).to.match(/loading.+retrospective/i)
-    } */)
+      return this.invoke(['-r'], this.notify, {lgJWT, lgPlayer}).then(() => {
+        expect(this.notifications).to.match(/this is the question body/i)
+        expect(this.notifications).to.match(/this is the second question body/i)
+        expect(this.notifications).to.not.match(/these are the instructions/i)
```

Comment:
Yeah, it works. I was a little surprised by this as well. I thought I was just copying the existing usage but now that I look at the diff it looks like I introduced this myself. Maybe I should change it to be less magical?